Sonic/System/GameParameter.h

```diff
@@ -0,0 +1,106 @@
+#pragma once
+
+#include <Hedgehog/Base/Type/hhSharedString.h>
+namespace Sonic
+{
+	class CGameParameter : public Hedgehog::Universe::CMessageActor
+	{
+	public:
+		struct SField7C
+		{
+		public:
+			BB_INSERT_PADDING(0x10);
+			byte m_Field10;
+			byte m_Field11;
+			byte m_Field12;
+			byte m_Field13;
+			int m_Field14;
+			int m_Field18;
+			BB_INSERT_PADDING(0xA0E0);
+			int m_FieldA0FC;
+			BB_INSERT_PADDING(0x120);
+			BB_INSERT_PADDING(0x80);
+			BB_INSERT_PADDING(0x20);
+			int m_FieldA2C0;
+			int m_FieldA2C4;
+		};
+		struct SStageParameters
+		{
+		public:
+			int m_Field00;
+			int m_Field04;
+			int m_Field08;
+			int m_Field0C;
+			int m_Field10;
+			hh::list<int> m_Field14;
+			Hedgehog::Base::CSharedString m_TerrainArchiveName;
+			Hedgehog::Base::CSharedString m_ModeName;
+			int m_Field28;
+			Hedgehog::Base::CSharedString m_Field2C;
+			Hedgehog::Base::CSharedString m_Field30;
+			int m_Field34;
+			int m_Field38;
+			int m_Field3C;
+			int m_Field40;
+			BB_INSERT_PADDING(0x18);
+			int m_Field5C;
+			int m_Field60;
+			BB_INSERT_PADDING(0x134);
+			int m_Field198;
+			int m_Field19C;
+			int* m_MissionFlags;
+			int m_Field1A4;
+			int m_Field1A8;
+			int m_Field1AC;
+			Hedgehog::Base::CSharedString m_BGMCueNodeName;
+			int m_Field1B4;
+			int m_Field1B8;
+			int m_Field1BC;
+
+		};
+		SField7C* m_Field7C;
```

Comment:
Need the p prefix here since it's a pointer.

Reply:
Fixed